corehq/apps/couch_sql_migration/management/commands/migrate_domain_from_couch_to_sql.py

```diff
@@ -198,16 +194,6 @@ def _confirm(message):
         raise CommandError('abort')
 
 
-def _get_sigterm_handler(domain):
-    def _sigterm_handler(signal, frame):
-        _logger.error("{} signal received".format(signal))
-        set_couch_sql_migration_not_started(domain)
-        clear_local_domain_sql_backend_override(domain)
-        _blow_away_migration(domain)
```

Comment:
Is there a way to manually initiate this cleanup? If not, does it matter if this is not done?

Another way of thinking about it is, does this leave the migration command in an unusable state? If yes, maybe better to roll this change in with whatever will be done to make it usable again.

Reply:
Yup. You can also just run with [`--blow-away`](https://github.com/dimagi/commcare-hq/blob/94b5606facaf07c8c5c69278e442ca85fe4dfe6e/corehq/apps/couch_sql_migration/management/commands/migrate_domain_from_couch_to_sql.py#L75-L83) independently. I added this to make it automatic which was nice at the time, but isn't worth it now